build.properties

```diff
@@ -1,3 +1,8 @@
 DOCKER_IMAGE_REPOSITORY=alfresco-base-tomcat
-DOCKER_IMAGE_TAG=8.5.28-java-8-oracle-centos-7
+DOCKER_IMAGE_TAG=8.5.34-java-8-oracle-centos-7
 DOCKER_IMAGE_TAG_SHORT_NAME=8.5
+
+
+TOMCAT_MAJOR=8
+TOMCAT_VERSION=8.5.34
+TOMCAT_SHA512=131dfe23918f33fb24cefa7a03286c786304151f95f7bc0b6e34dfb6b0d1e65fe606e48b85c60c8a522938d1a01a36b540e69c94f36973321858e229731cda82
```

Comment:
We'll need to update the SHA and version in 2 spots now?

Reply:
No... I'll remove the changes from build.properties. Oops.